classes/Infrastructure/Event/ExceptionListener.php

```diff
@@ -46,40 +45,25 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         $exception = $event->getException();
         $request   = $event->getRequest();
 
-        $event->setResponse($this->renderResponse($request, $exception));
-    }
-
-    private function renderResponse(Request $request, \Throwable $exception): Response
-    {
-        if (\in_array('application/json', $request->getAcceptableContentTypes())) {
-            $headers = [];
-
-            if ($exception instanceof HttpExceptionInterface) {
-                $code    = $exception->getStatusCode();
-                $headers = $exception->getHeaders();
-            }
-
-            return new JsonResponse([
-                'error' => $exception->getMessage(),
-            ], $code, $headers);
+        if (!\in_array('application/json', $request->getAcceptableContentTypes())) {
+            return;
         }
 
-        $template = 'error/500.twig';
-
-        $templates = [
-            Response::HTTP_UNAUTHORIZED => 'error/401.twig',
-            Response::HTTP_FORBIDDEN    => 'error/403.twig',
-            Response::HTTP_NOT_FOUND    => 'error/404.twig',
-        ];
+        $event->setResponse($this->renderResponse($exception));
+    }
 
-        $code = $exception instanceof HttpExceptionInterface ? $exception->getStatusCode() : 500;
+    private function renderResponse(\Throwable $exception): Response
+    {
+        $code    = 500;
```

Comment:
While it hasn't been used before, can we take the opportunity here to use the `HttpFoundation\Response::HTTP_INTERNAL_SERVER_ERROR` constant instead?